### PR TITLE
Add RemoveGraph action to Arrow Server

### PR DESCRIPTION
**Pull request by s1ck**
- Add RemoveGraph action
- Use RemoveGraph action in page-rank example
- Change Action descriptions

### CODE REVIEW

File: crates/server/src/catalog.rs [hunk 1]
```diff
@@ -187,6 +187,24 @@ impl GraphCatalog {
             })
             .collect::<Vec<_>>()
     }
+
+    pub fn remove<K: AsRef<str>>(&mut self, graph_name: K) -> Result<GraphInfo, Status> {
+        self.graphs.remove(graph_name.as_ref()).map_or_else(
+            || {
+                Err(Status::not_found(
+                    "Graph with name '{graph_name}' not found",
```
Comment (knutwalker) [inline crates/server/src/catalog.rs:195]: Is this missing a format?

```suggestion
                    format!("Graph with name '{graph_name}' not found"),
```

Comment (s1ck) [reply crates/server/src/catalog.rs:195]: It is .. and not just there 🙈

**Comment by s1ck** [comment]
@knutwalker 🏓

**Review by knutwalker — APPROVED** [review]
noice

**Pull request merged**